pubsub/azure/eventhubs/eventhubs.go

```diff
@@ -7,38 +7,48 @@ package eventhubs
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"strconv"
+	"strings"
 	"time"
 
+	"github.com/Azure/azure-amqp-common-go/v3/aad"
+	"github.com/Azure/azure-amqp-common-go/v3/conn"
 	eventhub "github.com/Azure/azure-event-hubs-go/v3"
 	"github.com/Azure/azure-event-hubs-go/v3/eph"
```

Comment:
use of event processor host will also affect batch API ... as right now there does not seem to be any option for batching within EPH.